### PR TITLE
[14.0][IMP][barcodes_generator_product] multi barcodes generation

### PR DESCRIPTION
**Pull request by Kev-Roche**
Add an action on products to generate barcodes for multiple products.
Also add a filter for products without barcode. 
cc @legalsylvain

### CODE REVIEW

File: barcodes_generator_product/wizard/multi_barcode_wizard.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <form string=" ">
+                <div
+                    class="alert alert-danger alert-dismissable"
+                    role="alert"
+                    style="margin-bottom:0px;"
+                    attrs="{'invisible':[('product_barcode_exception','=', False)]}"
+                >
+                    <field name="product_barcode_exception" />
+                </div>
+                <sheet>
```
Comment (legalsylvain) [inline barcodes_generator_product/wizard/multi_barcode_wizard.xml:19]: If you add product_ids on the view, users can add products.

File: barcodes_generator_product/models/product_product.py [hunk 2]
```diff
@@ -9,3 +9,18 @@
...
     _name = "product.product"
     _inherit = ["product.product", "barcode.generate.mixin"]
+
+    def show_generate_multi_barcodes(self):
+        view_id = self.env.ref(
+            "barcodes_generator_product.view_generate_multi_barcodes_form"
+        ).id
+        wizard = self.env["barcode.multiple.generator.wizard"].create({})
+        wizard.product_ids = self.env.context["active_ids"]
+        return {
```
Comment (legalsylvain) [inline barcodes_generator_product/models/product_product.py:23]: You can filter here products without barcode.

File: barcodes_generator_product/wizard/multi_barcode_wizard.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    generate_type = fields.Selection(
+        string="Generate Type",
+        related="barcode_rule_id.generate_type",
+        readonly=True,
+    )
+    product_barcode_exception = fields.Html(
+        readonly=True, compute="_compute_product_barcode_exception"
+    )
+
+    @api.depends("product_ids", "product_ids.barcode", "barcode_rule_id")
```
Comment (legalsylvain) [inline barcodes_generator_product/wizard/multi_barcode_wizard.py]:
```suggestion
    @api.depends("product_ids.barcode")
```

Is enought dont you think ?

**Review by legalsylvain — COMMENTED** [review]
Hi.

Thanks for you contribution !

Some minor remarks inline.

**Comment by Kev-Roche** [comment]
hi @legalsylvain I apply your suggestion

**Pull request closed without merging**